src/Infrastructure.Dapper/AdminConsole/Repositories/OrganizationRepository.cs

```diff
@@ -251,4 +262,103 @@ await connection.ExecuteAsync("[dbo].[Organization_IncrementSeatCount]",
+            occupiedSeatCount.Total,
+            organizationUsersList.Count);
+
+        var result = InviteUsersPasswordManagerValidator.ValidatePasswordManager(validatorRequest);
+
+        switch (result)
+        {
+            case Valid<PasswordManagerSubscriptionUpdate> valid:
+                try
+                {
+                    await connection.ExecuteAsync(
+                        $"[{Schema}].[OrganizationUser_CreateManyWithCollectionsAndGroups]",
+                        new
+                        {
+                            OrganizationUserData =
+                                JsonSerializer.Serialize(organizationUsersList.Select(x => x.OrganizationUser)),
+                            CollectionData = JsonSerializer.Serialize(organizationUsersList
+                                .SelectMany(x => x.Collections,
+                                    (user, collection) => new CollectionUser
+                                    {
+                                        CollectionId = collection.Id,
+                                        OrganizationUserId = user.OrganizationUser.Id,
+                                        ReadOnly = collection.ReadOnly,
+                                        HidePasswords = collection.HidePasswords,
+                                        Manage = collection.Manage
+                                    })),
+                            GroupData = JsonSerializer.Serialize(organizationUsersList
+                                .SelectMany(x => x.Groups,
+                                    (user, group) => new GroupUser
+                                    {
+                                        GroupId = group,
+                                        OrganizationUserId = user.OrganizationUser.Id
+                                    }))
+                        },
+                        commandType: CommandType.StoredProcedure);
+
+                    if (valid.Value.SeatsRequiredToAdd > 0)
+                    {
+                        await connection.ExecuteAsync("[dbo].[Organization_IncrementSeatCount]",
+                            new
+                            {
+                                OrganizationId = organizationId,
+                                SeatsToAdd = valid.Value.SeatsRequiredToAdd,
+                                RequestDate = requestDate
+                            },
+                            commandType: CommandType.StoredProcedure);
+                    }
+
+                    await transaction.CommitAsync();
+                }
+                catch (Exception)
+                {
+                    await transaction.RollbackAsync();
+                    throw;
+                }
+
+                break;
+            case Invalid<PasswordManagerSubscriptionUpdate> invalid:
+                await transaction.RollbackAsync();
+                throw new Exception(invalid.Error.Message);
+            default:
```

Comment:
❓ Per my other comment, does anything need to happen? Is this actually worthy of throwing? Can't just throw the base exception either.